cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,18 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
+    cy.visit('https://react-redux.realworld.io');
 
+    cy.contains('Sign in').click();
```

Comment:
For this task, it is enough, but for the future, I would recommend writing more unique ways to the element. Good job overall.